server/controllers/chartsController/index.js

```diff
@@ -0,0 +1,70 @@
+import { ObjectID } from 'mongodb'
+
+import { collections } from '../../utils/mongoCollections'
+import {
+  generateStudyTargetTotals,
+  isAnyTargetIncluded,
+  N_A,
+  postProcessData,
+  processData,
+  processTotals,
+  TOTALS_STUDY,
+} from './helpers'
+
+const STUDIES_TO_OMIT = ['files', 'combined']
```

Comment:
can you help me understand why this const is not part of the helpers file?

Reply:
yup, this constant is defined here because it is only used in this file. If it was used in the helper file then it would be defined there to avoid a cyclical import.